src/Propel/Generator/Behavior/Archivable/ArchivableBehaviorQueryBuilderModifier.php

```diff
@@ -59,6 +84,11 @@ public function preDeleteQuery($builder)
         }
     }
 
+    /**
+     * @param \Propel\Generator\Builder\Om\AbstractOMBuilder $builder
+     *
+     * @return string|null
+     */
     public function postUpdateQuery($builder)
```

Comment:
No action needed, (I just want to share some thoughts I had). It will be cleaner for types when we make these return empty strings instead of nulls. We also get to execute half the if-statements for the same result.

Reply:
I agree that a future cleanup pr should normalize to string only.